english/Booyah/booyah.md

```diff
@@ -4,3 +4,4 @@ What am I going to do with my new found knowledge?
 The world is now my [oyster.](https://en.wikipedia.org/wiki/Pacific_oyster)
 
 Have some booyah stew
+or call a [spirit](call-spirit/call-spirit.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.